test/multest.jl

```diff
@@ -11,6 +11,7 @@ qv = QuArray(v)
 @assert rawcoeffs(qm'*qm') == m*m
```

Comment:
`rawcoeffs` and `coeffs` return the same for all the cases except for the recent commit. @acroy please do let me know if am missing something further here.